astro_2.0/features/ViewProposal/components/ProposalComments/helpers.tsx

```diff
@@ -0,0 +1,35 @@
+import cn from 'classnames';
```

Comment:
This are actually components, not helpers and I consider we should place them in proper files.

Reply:
It is already updated, I removed this file